index.js

```diff
@@ -1,12 +1,13 @@
-var events = require('./events')(require('./v3'))
-var Stream = require('./stream')(events)
 var progress = require('./progress')
 
 function timestamp () {
   return Date.now()
 }
 
 module.exports = function (opts) {
+  var events = require('./events')(require('./v3'))
+  var Stream = require('./stream')(events)
```

Comment:
Just curious: what does this fix? (The next naive programmer to come along might move them back)

Reply:
Right, this was not the best change. I'll do another one and add proper description. It was a bit late yesterday ;)